python/CRUD-examples/reset_data.py

```diff
@@ -0,0 +1,36 @@
+#code Sample from the tutorial at https://learncodeshare.net/2015/06/02/basic-crud-operations-using-cx_oracle/
+#The following resets the data to the initial state for the tutorial.
+
+import cx_Oracle
+import os
+connectString = os.getenv('db_connect')
+con = cx_Oracle.connect(connectString)
+cur = con.cursor()
+
+# Delete rows
+statement = 'delete from lcs_pets'
+cur.execute(statement)
+
+# Delete rows
+statement = 'delete from lcs_people'
+cur.execute(statement)
+
+# Reset Identity Coulmn
+statement = 'alter table lcs_people modify id generated BY DEFAULT as identity (START WITH 3)'
+cur.execute(statement)
+
+# Insert default rows
+rows = [(1, 'Bob', 35, 'I like dogs'), (2, 'Kim', 27, 'I like birds')]
+cur.bindarraysize = 2

```

Comment:
The reset_data modules are included in the tutorial as utility/helper functions.  The functionality used in these modules is explained in tutorial under the related sections.  For example, the executemany functionality is covered in the insert section under "Insert more than 1 row".